docs/source/index.rst

```diff
@@ -16,6 +16,7 @@ Here you will find all the relevant information about the project.
    Guides <guides/guides>
    Tutorials <tutorials/tutorials>
    Advanced Utilities <advanced_utils>
+   Migration Guide <migration_guide/migration_guide>
```

Comment:
Migration Guide**s**

We can call this tree section more generic so we can use it later on.